src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -225,15 +225,6 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
     `;
   }
 
-  private _stateIconLabel(state: string): string {
-    const stateLabel = state.split("_").pop();
-    return stateLabel === "disarmed" ||
-      stateLabel === "triggered" ||
-      !stateLabel
-      ? ""
```

Comment:
You now also changed the logic of not showing the state on the badge when it is `disarmed` or `triggered`.

Reply:
In my second screenshot down you can see the badge in the disarmed state so I'm not sure this logic was being used any more?

I reinstated using your suggestion below swapping `: stateLabel;` to `this._stateDisplay(state)` and tested and this stopped text rendering again until it was borked to _disrmed_ at which point it skipped it and rendered the translation correctly using the amended `this._stateDisplay(state)`

I slimmed down to this:

```
  private _stateIconLabel(state: string): string {return this._stateDisplay(state);
  }

```
And it worked for both of these states:

![image](https://user-images.githubusercontent.com/54028700/81478942-4e5fe800-9218-11ea-92ce-c7b829c14de6.png)

![image](https://user-images.githubusercontent.com/54028700/81479152-8e739a80-9219-11ea-98ab-573f2008f748.png)

Reply:
But in the original code, there is no text in the badge when the state is unarmed.

Reply:
![image](https://user-images.githubusercontent.com/5662298/81482839-3a2ce280-923a-11ea-8ddc-f85d24d6a3dd.png)

Reply:
Oh yes, d'oh. I got so wrapped up in the translations, I missed that!